lib/account.js

```diff
@@ -162,6 +162,101 @@ module.exports = function(config) {
           callback(err, json);
         });
       });
+    },
+    requestMigrateEmail: function(request, callback) {
+      var appURLObj = url.parse(config.uri + '/migrate', true);
+      appURLObj.query = request.payload.oauth;
+
+      var migrateRequest = hyperquest({
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          'x-ratelimit-ip': getIPAddress(request)
+        },
+        uri: loginAPI + '/api/v2/user/request'
+      });
+
+      migrateRequest.on('error', callback);
```

Comment:
https://www.npmjs.com/package/hyperquest#req-on-error-function-res

If I understand that correctly, if an http request times out, it will be passed via the error event.